eng/pipelines/templates/steps/npm-release-task.yml

```diff
@@ -0,0 +1,55 @@
+parameters:
+  ArtifactName: ''
+  Artifact: {}
+  Registry: ''
+  PathToArtifacts: ''
+  Tag: ''
+  AdditionalTag: ''
+
+steps:
+  - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
+  - ${{if eq(parameters.Registry, 'https://registry.npmjs.org/')}}:
+      - task: PowerShell@2
+        displayName: Create publish package directory
+        inputs:
+          targetType: filePath
+          filePath: eng/tools/publish-to-npm.ps1
+          arguments: -pathToArtifacts ${{parameters.PathToArtifacts}} -accessLevel "public" -tag "${{parameters.Tag}}" -registry ${{parameters.Registry}} -npmToken $(azure-sdk-npm-token) -packagesToPublishPath "$(Pipeline.Workspace)/temp/${{parameters.ArtifactName}}"
+          pwsh: true
+        condition: succeeded()
+      - task: EsrpRelease@6
```

Comment:
So we publish to npm ourselves AND through esrp?

Reply:
the task above it is just prepping the package (determinig which one and copy it to a folder). then esrp releases the folder

Reply:
Should the script be renamed? Is it used anywhere to actually publish?

Reply:
yes, it's used for dev feed publishing, the plan is to have better separation later on when the change tag feature is in place from the esrp